src/Stratis.Bitcoin/P2P/PeerAddressManager.cs

```diff
@@ -103,10 +90,14 @@ public sealed class PeerAddressManager : IPeerAddressManager
         /// <inheritdoc />
         public DataFolder PeerFilePath { get; set; }
 
+        /// <inheritdoc />
+        public IPeerSelector Selector { get; private set; }
```

Comment:
Its easier if you call the param `PeerSelector`

Reply:
Will rename 👍